specification/network/resource-manager/Microsoft.Network/stable/2023-11-01/networkVirtualAppliance.json

```diff
@@ -285,6 +285,67 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkVirtualAppliances/{networkVirtualApplianceName}/restart": {
```

Comment:
Please fix Model Validation errors shown in PR: [https://github.com/Azure/azure-rest-api-specs/pull/27467/checks?check_run_id=20746953241](https://github.com/Azure/azure-rest-api-specs/pull/27467/checks?check_run_id=20746953241).

Reply:
Fixed